src/draive/gemini/config.py

```
@@ -9,8 +9,21 @@
     "GeminiEmbeddingConfig",
     "GeminiGenerationConfig",
     "GeminiLiveConfig",
+    "GeminiSafetyConfig",
+    "GeminiSafetyThreshold",
 )
 
+GeminiSafetyThreshold = Literal["off", "none", "high", "medium", "low"]
+
+
+class GeminiSafetyConfig(Config):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)